chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/search/CertifiedProductBasicSearchResult.java

```diff
@@ -5,7 +5,7 @@
 
 import com.fasterxml.jackson.annotation.JsonView;
 
-public class CertifiedProductBasicSearchResult extends CertifiedProductSearchResult {
+public class CertifiedProductBasicSearchResult extends CertifiedProductSearchResultLegacy {
```

Comment:
Should we be extending a `@Deprecated` class?  I'm not sure...  Are we still using the `CertifiedProductSearchResultLegacy` ? Should `CertifiedProductSearchResultLegacy` maybe be abstract? I'm not sure what the right answer is here off the top of my head.